### DISCUSSION TITLE
Selection of map storage directory

### DISCUSSION THREAD

**Discussion by tuxianerDE** [organicmaps/organicmaps#1350] [Ideas]
As a user I would like to select the map storage directory. This becomes in handy especially if multiple applications can access the maps and directory.

**Comment by biodranik** [comment]
Which multiple applications do you want to use for the same maps? What is your use case?

**Comment by tuxianerDE** [comment]
I am using c:geo for geocaching and like to make use of the offline map capability.

Cheers
